reek.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |s|
   s.summary = 'Code smell detector for Ruby'
 
   s.add_runtime_dependency 'codeclimate-engine-rb', '~> 0.3.1'
-  s.add_runtime_dependency 'parser',                '~> 2.3', '>= 2.3.0.6'
+  s.add_runtime_dependency 'parser',                '~> 2.3.1'
```

Comment:
Let's ensure that at least `2.3.1.1` is used since it includes the fix for [this safe navigation issue](https://github.com/troessner/reek/issues/836#issuecomment-222181329).